cpp/src/randomforest/randomforest_impl.cuh

```diff
@@ -221,10 +224,10 @@ void rfClassifier<T>::fit(const cumlHandle& user_handle, const T* input,
     unsigned int* rowids;
     rowids = selected_rows[stream_id]->data();
 
-    this->prepare_fit_per_tree(i, n_rows, n_sampled_rows, rowids,
-                               tempmem[stream_id]->num_sms,
-                               tempmem[stream_id]->stream,
-                               handle.getDeviceAllocator());
+    this->prepare_fit_per_tree(
+      i, n_rows, n_sampled_rows, rowids, (this->rf_params.seed + i),
```

Comment:
Can we do the seed increment inside the function and not as function argument. so only have to maintain at one place

Reply:
Makes sense. Updated it.